### PR TITLE
Issue3 introduction general

### PR DESCRIPTION
**Pull request by niellaNina**
Ready for review, is this general enough

### CODE REVIEW

File: doc/introduction.adoc [hunk 1]
```diff
@@ -1,4 +1,218 @@
...
+3. to provide personnel with practical guidelines for carrying out good data management.
+
+_Data management_ is the term used to describe the handling of data in a systematic and cost-effective manner. 
+The data management regime should be continuously evolving, to reflect the evolving nature of data colletion. Therefore this DMH is a living document that will be revised and updated from time to time in order to maintain its relevance.
+
+The DMH is a strategic governing document and should be used as part of the quality framework the organisation is using. 
+
+// Remember to add links/references to the chapters below
+
+This DMH focuses on the management of geophysical data, types of data and the usage of these data are described in the organisation specific information in Section X.X.X.
```
Comment (ElodieFZ) [inline doc/introduction.adoc:21]: I think something is odd with this sentence. Maybe split it in two?

```suggestion
This DMH focuses on the management of geophysical data. Types of data and the usage of these data are described in the organisation specific information in Section X.X.X.
```

File: doc/introduction.adoc [hunk 2]
```diff
@@ -1,4 +1,218 @@
...
+The Quick Manual is a concise, informal HOW-TO for data management practitioners.
+
+//Remember to change if we move away from the Quick manual
+
+The intended audience for this DMH is any co-worker who is involved in any part of the value chain for geophysical data (<<img-value_chain>>).
+
+The handbook can be used in three ways: 
+
+1. Read the Introduction (Chapter 1) to find out the background and principles of data management;
+2. Read Chapters 2-5 to learn about how data management is currently implemented and how it is expected to evolve in the next few years:
```
Comment (ElodieFZ) [inline doc/introduction.adoc:35]:
```suggestion
2. Read Chapters 2-5 to learn about how data management is currently implemented and how it is expected to evolve in the next few years;
```

File: doc/introduction.adoc [hunk 3]
```diff
@@ -1,4 +1,218 @@
...
+3. Go to the Quick Manual for Data Providers for guidelines on what to do in real-life cases; alternatively Chapter 6 for typical workflow examples.
```
Comment (ElodieFZ) [inline doc/introduction.adoc:36]:
```suggestion
3. Read the Quick Manual for Data Providers for guidelines on what to do in real-life cases; alternatively read Chapter 6 for typical workflow examples.
```

File: doc/introduction.adoc [hunk 4]
```diff
@@ -1,4 +1,218 @@
...
+
+* the ease of discovering, viewing and accessing all the datasets that are offered by the institute;
+* standardised ways of downloading data, which reduces the need for special solutions on the user side;
+* reducing their own data storage needs, by downloading just what they need;
+* easy and standardised access to remote datasets and catalogues, when using their own visualisation/analysis tools;
+* the ability to compare and combine MET Norway’s data with data from other sources (through metadata catalogues);
+* the ability to apply common data transformations, like spatial, temporal and variables subsetting and reprojection, before downloading anything;
+*the ease of building specific metadata catalogues and data portals that include data from MET Norway and can target a specific user community;
```
Comment (ElodieFZ) [inline doc/introduction.adoc:49]:
```suggestion
* the ease of building specific metadata catalogues and data portals that include data from MET Norway and can target a specific user community;
```

Comment (ElodieFZ) [inline doc/introduction.adoc:49]:
```suggestion
*the ease of building specific metadata catalogues and data portals that include data from the institute and can target a specific user community;
```

File: doc/introduction.adoc [hunk 5]
```diff
@@ -1,4 +1,218 @@
...
+
+At the end of the data management value chain are the users of the data (aka. data consumers, see Section 1.2.2.1), who may be either external or internal to the institute.
+
+[[fair-data-management-model]]
+==== A data management model based on the FAIR principles
+
+This model is based on the model of the Arctic Data Centre, which adheres to the FAIR principles. The model’s basic functions fall into three main categories:
+
+1. *Documentation of data* using discovery and use metadata (metadata are further described below). 
+The documentation identifies who, what, when, where, and how and shall make it easy for consumers to find and understand data. This requires application of information containers and utilisation of controlled vocabularies and ontologies where textual representation is required. It also covers the topic of data provenance which is used to describe the origin and all actions done on a dataset. Data provenance is closely linked with workflow management. Furthermore, it covers the relationship between datasets. Application of ontologies in data documentation is closely linked to the concept of linked data.
```
Comment (ElodieFZ) [inline doc/introduction.adoc:86]:
```suggestion
The documentation identifies who, what, when, where, and how, and shall make it easy for consumers to find and understand data. This requires application of information containers and utilisation of controlled vocabularies and ontologies where textual representation is required. It also covers the topic of data provenance which is used to describe the origin and all actions done on a dataset. Data provenance is closely linked with workflow management. Furthermore, it covers the relationship between datasets. Application of ontologies in data documentation is closely linked to the concept of linked data. 
```

File: doc/introduction.adoc [hunk 6]
```diff
@@ -1,4 +1,218 @@
...
+
+|[[use-metadata]]Use metadata
+|Used to understand data found
+|Use metadata describe the actual content of a dataset and how it is encoded. The purpose is to enable the user to understand the data without any further communication. They describe the content of variables using standardised vocabularies, units of variable, encoding of missing values, map projections, etc.
+|Climate and Forecast (CF) Convention
+BUFR
+GRIB
+
+|[[configuration-metadata]]Configuration metadata
+|Used to tune portal services for datasets for users.
```
Comment (ElodieFZ) [inline doc/introduction.adoc:141]:
```suggestion
|Used to tune portal services for datasets for users
```

File: doc/introduction.adoc [hunk 7]
```diff
@@ -1,4 +1,218 @@
...
+|[[site-metadata]]Site metadata
+|Used to understand data found
+|Site metadata are used to describe the context of observational data. They describe the location of an observation, the instrumentation, procedures, etc. To a certain extent they overlap with discovery metadata, but also extend the discovery metadata. Site metadata can be used for observation network design. Site metadata can be considered a type of use metadata.
+|WIGOS
+OGC O&M
+StInfoSys
+
+|[[system-metadata]]System metadata
+|Used to understand the technical structure of the data management system and track changes in it 
+|System metadata covers e.g. technical details of the storage system (e.g. Lustre metadata), web services their purpose and how they interact with other components of the data management system, available and consumed storage, number of users and other KPI elements etc.
```
Comment (ElodieFZ) [inline doc/introduction.adoc:154]:
```suggestion
|System metadata covers e.g. technical details of the storage system (e.g. Lustre metadata), web services, their purpose and how they interact with other components of the data management system, available and consumed storage, number of users and other KPI elements etc.
```

File: doc/introduction.adoc [hunk 8]
```diff
@@ -1,4 +1,218 @@
...
+|Climate and Forecast (CF) Convention
+BUFR
+GRIB
+
+|[[configuration-metadata]]Configuration metadata
+|Used to tune portal services for datasets for users.
+|Configuration metadata are used to improve the services offered through a portal to the user community. This can be e.g. how to best visualise a product.
+|
+ 
+|[[site-metadata]]Site metadata
```
Comment (ElodieFZ) [inline doc/introduction.adoc:145]: I would put site metadata just after use metadata as they are kind of related. Then put the configuration metadata and system metadata.

**Review by ElodieFZ — CHANGES_REQUESTED** [review]
Nice work! I seems to me it's easier to read now, there are less big chunks of text. I review from the pdf, so I made comments on both the text and how it looks like in the final pdf.

File: doc/glossary.adoc [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+[%header, cols=2*]
+|===
+|Term
+|Description
+
+|[[application-service]]Application service
+|TBC
+
+|[[CMD-dataset]]CDM dataset
+|A dataset that “may be a NetCDF, HDF5, GRIB, etc. file, an OPeNDAP dataset, a collection of files, or anything else which can be accessed through the NetCDF API.” https://www.unidata.ucar.edu/software/netcdf-java/v4.6/CDM/index.html[Unidata Common Data Model]
```
[the remark this answers is not in the document]
Comment (niellaNina) [reply doc/glossary.adoc:16]: Added this to the list of things to check in the next issue

File: doc/glossary.adoc [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+|[[fair-principles]]FAIR principles
+|The four foundational principles of good data management and stewardship: *F*indability, *A*ccessibility, *I*nteroperability and *R*eusability. Nature article [https://www.nature.com/articles/sdata201618[RD3]], https://www.go-fair.org/fair-principles/[FAIR Data Principles], https://github.com/FAIRMetrics/Metrics[FAIR metrics proposal], https://ec.europa.eu/research/participants/data/ref/h2020/grants_manual/hi/oa_pilot/h2020-hi-oa-data-mgt_en.pdf[EU H2020 Guidelines]
+
+|[[feature-type]]Feature type
+|A categorisation of data according to how they are stored, for example, grid, time series, profile, etc. It has been formalised in the NetCDF/CF https://www.nodc.noaa.gov/data/formats/netcdf/v2.0/#templatesexamples[feature type table], which currently defines eight feature types. 
+//need internal link to acronym-document
+
+|[[geodataloven]]Geodataloven
+|“Norwegian regulation toward good and efficient access to public geographic information for public and private purposes.” See https://www.regjeringen.no/no/tema/plan-bygg-og-eiendom/plan--og-bygningsloven/kart/geodataloven/id749728/[Deling av geodata – Geodataloven].
+
```
[the remark this answers is not in the document]
Comment (niellaNina) [reply doc/glossary.adoc:71]: I will look into this

File: doc/introduction.adoc (hunk 8 above)
Comment (niellaNina) [reply doc/introduction.adoc:145]: I will move this in the next issue

**Comment by niellaNina** [comment]
The more deep changes will be moved to a new issue

**Pull request merged**